### PR TITLE
Change in UI

### PR DESCRIPTION
**Pull request by cryogon**
I made some changes in UI because the old one doesn't look good but still it is in the test phase and ui will be changed later

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  computed: {
+    ...mapState(["visible"]),
+  },
+};
+</script>
+<style lang="scss">
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@200&family=Montserrat:wght@200&display=swap");
+body {
+  font-family: "Inter", sans-serif;
+  // background-color: #67e0a3;
```
Comment (blahmonkey) [inline src/App.vue]: @JatinThakur14 Dont need to change this now, but in future, generally remove commented code when you push commits. The code is anyway readable from git version history

File: src/components/HelloWorld.vue [hunk 2]
```diff
@@ -0,0 +1,130 @@
+<template>
```
Comment (blahmonkey) [inline src/components/HelloWorld.vue]: @JatinThakur14 I think this file isnt used any more?

File: src/components/LoginForm.vue [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+      <button @click.prevent="doLogin">Login</button>
+    </form>
+  </div>
+</template>
+
+<script>
+export default {
+  methods: {
+    doLogin() {
+      console.log("klik");
```
Comment (blahmonkey) [inline src/components/LoginForm.vue]: @JatinThakur14 clean up this console log, it isnt needed now

File: src/router/index.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  },
+];
+
+const router = createRouter({
+  history: createWebHashHistory(),
+  routes,
+});
+
+router.beforeEach((to, from, next) => {
+  console.log(store.getters.storeToken);
```
Comment (blahmonkey) [inline src/router/index.js]: @JatinThakur14 Clean up this console log, its not needed

**Review by blahmonkey — CHANGES_REQUESTED** [review]

**Comment by cryogon** [comment]
I removed comments from file and also deleted HelloWorld.vue file

**Review by blahmonkey — APPROVED** [review]

**Pull request merged**